myhpi/core/templates/core/minutes.html

```diff
@@ -8,69 +8,74 @@
     <div class="row minutes-container">
         {% with page.body|markdown as parsed_md %}
         <div class="d-none d-print-block minutes-title">
-        <h1>{{  page.title }}</h1>
+            <h1>{{  page.title }}</h1>
         </div>
         <div class="col-9 minutes-text">
             {{ parsed_md.0 }}
         </div>
         <div class="col-3 minutes-meta">
-            <h4>{% translate "Date" %}</h4>
-            <p>{{ page.date }}</p>
-            <h4>{% translate "Moderator" %}</h4>
-            <p>{{ page.moderator.get_full_name }}</p>
-            <h4>{% translate "Minutes taker" %}</h4>
-            <p>{{ page.author.get_full_name }}</p>
-            <h4>{% translate "Participants" %}</h4>
-            <ul>
-                {% for participant in page.participants.all %}
-                    <li>{{ participant.get_full_name }}</li>
-                {% endfor %}
-            </ul>
-            <h4>{% translate "Guests" %}</h4>
-            {% if page.guests %}
+            {% if request.is_preview %}
```

Comment:
this labels previews, in the list we are labelling drafts. Is this intentional?

Reply:
You are right, updated.

Although I am wondering how they differ: Does `page.live` mean that it only shows this label if there is no version published?

Reply:
see https://docs.wagtail.org/en/stable/reference/pages/model_reference.html#wagtail.models.Page.live

after reading this, checking for `page.has_unpublished_changes` probably is what we want

Reply:
Hm, according to the page
> A boolean that is set to True when the page is either in draft or published with draft changes.
this also shows if the page is published, but there's a drafted version. I just checked and it indeed also shows the label on the published page if there is a draft.

Reply:
`page.live` indeed only shows the label if there is _no_ other published version of the page...

Reply:
but isn't having unpublished changes exactly what we want to protect against? I remember that we had that one minutes that was published nearly empty (but visible for noone) and all of the meaningful content was in a draft

Reply:
Hm. In this case, we would also need to check the user's permission to change the page (since otherwise, the label would be displayed to everyone). I'll look into that soon.

Reply:
I think you are doing the check already to display the page visibility

Reply:
Updated.